src/utils/tests/dispatcher.test.js

```diff
@@ -26,19 +31,19 @@ describe('parseMessage', () => {
 describe('dispatchCmd', () => {
   test('dispatches event with empty array', () => {
     const emitterSpy = jest.spyOn(client, 'emit');
-    dispatchCmd('faq', []);
-    expect(emitterSpy).toHaveBeenCalledWith('faq', []);
+    dispatchCmd('faq', [], '++faq');
```

Comment:
is the last thing meant to be a string or message object?

Reply:
Message object, but afaik all that matters is the toString version, which is done for us here

Reply:
Ah this is why TS would be nice. Let's make this test work the way it should and pass a message object here